project/SparkBuild.scala

```diff
@@ -155,14 +155,16 @@ object SparkBuild extends PomBuild {
   // We special case the 'println' lint rule to only be a warning on compile, because adding
   // printlns for debugging is a common use case and is easy to remember to remove.
   val scalaStyleOnCompileConfig: String = {
-    val in = "scalastyle-config.xml"
+    val base_style_config = "scalastyle-config.xml"
+    val prod_style_config = "scalastyle-config-prod.xml"
```

Comment:
1. Is this a convention? Does something named `-prod` only check the format of a specific directory?
2. It is necessary to synchronize the modifications to the scalastyle-maven-plugin in the parent pom.xml.

Reply:
The `scalastyle-config-prod.xml` is for production code, which means it won't check the style of tests.